docs/pages/chat-apps/list-stream-sync/list-messages.mdx

```diff
@@ -0,0 +1,510 @@
+    println("Reply text: ${replyContent.content}")
+  }
+}
+```
+
+:::
+
+## List messages (unenriched)
+
+Retrieve messages from a conversation.
+
+:::code-group
+
+```swift [Swift]
+// Get all messages
+let messages = try await group.messages()
+
+// Get the 10 most recent messages
+let recentMessages = try await group.messages(limit: 10)
+
+// Get messages after a specific time (e.g., for pagination or unread messages)
+let lastCheckTimestamp: Int64 = 1738620126404999936
+let newMessages = try await group.messages(afterNs: lastCheckTimestamp)
+
+// Get messages from a DM conversation
+let dm = try await client.conversations.findOrCreateDm(with: recipientInboxId)
+let dmMessages = try await dm.messages(limit: 50)
+```
+
+```kotlin [Kotlin]
+// Get all messages
+val messages = group.messages()
+
+// Get the 10 most recent messages
+val recentMessages = group.messages(limit = 10)
+
+// Get messages after a specific time (e.g., for pagination or unread messages)
+val lastCheckTimestamp: Long = 1738620126404999936
+val newMessages = group.messages(afterNs = lastCheckTimestamp)
+
+// Get messages from a DM conversation
+val dm = client.conversations.findOrCreateDm(recipientInboxId)
+val dmMessages = dm.messages(limit = 50)
+```
+
+```tsx [React Native]
+// Get all messages
+const messages = await group.messages();
+
+// Get the 10 most recent messages
+const recentMessages = await group.messages({ limit: 10 });
+
+// Get messages after a specific time (e.g., for pagination or unread messages)
+const lastCheckTimestamp = 1738620126404999936;
+const newMessages = await group.messages({
+  afterNs: lastCheckTimestamp,
+});
+
+// Get messages from a DM conversation
+const dm = await client.conversations.findOrCreateDm(recipientInboxId);
+const dmMessages = await dm.messages({ limit: 50 });
+```
+
+```tsx [Browser]
+// Get all messages
+const messages = await group.messages();
+
+// Get the 10 most recent messages
+const recentMessages = await group.messages({ limit: 10n });
+
+// Get messages after a specific time (e.g., for pagination or unread messages)
+const lastCheckTimestamp = 1738620126404999936n;
+const newMessages = await group.messages({
+  sentAfterNs: lastCheckTimestamp,
+});
+
+// Get messages from a DM conversation
+const dm = await client.conversations.findOrCreateDm(recipientInboxId);
+const dmMessages = await dm.messages({ limit: 50n });
+```
+
+```tsx [Node]
+// Get all messages
+const messages = await group.messages();
+
+// Get the 10 most recent messages
+const recentMessages = await group.messages({ limit: 10n });
+
+// Get messages after a specific time (e.g., for pagination or unread messages)
+const lastCheckTimestamp = 1738620126404999936n;
+const newMessages = await group.messages({
+  sentAfterNs: lastCheckTimestamp,
+});
+
+// Get messages from a DM conversation
+const dm = await client.conversations.findOrCreateDm(recipientInboxId);
+const dmMessages = await dm.messages({ limit: 50n });
+```
+
+:::
+
```

Comment:
count messages just afte rthe last timestamp

and another with the content types - mix and match filters.